frontend/src/scenes/instance/Licenses/index.tsx

```diff
@@ -8,66 +9,175 @@ import { PageHeader } from 'lib/components/PageHeader'
 import { InfoCircleOutlined } from '@ant-design/icons'
 import { Tooltip } from 'lib/components/Tooltip'
 import { SceneExport } from 'scenes/sceneTypes'
+import { LemonTable, LemonTableColumns } from 'lib/components/LemonTable'
+import { LicenseType, TeamType } from '~/types'
+import { LemonButton } from 'lib/components/LemonButton'
+import { organizationLogic } from 'scenes/organizationLogic'
+import { LemonModal } from 'lib/components/LemonModal'
+import { dayjs } from 'lib/dayjs'
 
 export const scene: SceneExport = {
     component: Licenses,
     logic: licenseLogic,
 }
 
-const columns = [
-    {
-        title: 'Active',
-        render: function renderActive(license: any) {
-            return isLicenseExpired(license) ? 'expired' : 'active'
+function ConfirmCancel({
+    licenses,
+    onCancel,
+    onOk,
+}: {
+    licenses: LicenseType[]
+    onCancel: () => void
+    onOk: () => void
+}): JSX.Element {
+    const { currentOrganization } = useValues(organizationLogic)
+    const hasAnotherValidLicense = licenses.filter((license) => dayjs().isBefore(license.valid_until)).length > 0
```

Comment:
This expression will almost always be incorrectly true here, because it just checks if the user has _a_ valid active license, which the one they are canceling is likely to be. That one should not be taken into account.